### DISCUSSION TITLE
【灌水】《致命营救》好看吗?

### DISCUSSION THREAD

**Discussion by nana26710** [share121/inter-knot#4738] [灌水]
最近大家都在讨论《致命营救》，我看评分网站评价两极分化，到底好看不好看啊
（搬运）
![Uploading IMG_20240909_201252.jpg…]()

**Comment by nana26710** [comment]
（图片没发出来QAQ）
![IMG_20240909_201252](https://github.com/user-attachments/assets/51807b87-6be3-48bd-855e-dd91992ce7a3)

**Comment by Robin-txdy** [comment]
没看过，蹲一个测评<img src="https://upload-bbs.mihoyo.com/upload/2022/09/09/709ea11ad3ec0af2101e3036568659df_4432728589989855604.png" alt="神里绫华-乖巧" width="75" >
